### DISCUSSION TITLE
Dynamic localization

### DISCUSSION THREAD

**Discussion by yury-github** [KevinVandy/material-react-table#141] [Q&A]
Hi, friends.
How i can dynamic change localization with i18n?

**Comment by KevinVandy** [comment]
localization is currently memoized internally, since I didn't anticipate anyone wanting that. Maybe the next patch will allow it. For now maybe adding a key prop for the selected locale will work for you?

**Reply by KevinVandy** [reply]
This should be fixed in `v1.2.6`

What language are you translating to btw? Consider making a PR to add it to the library!
